qcut/docs/task/e2e-test-plan.md

```
@@ -0,0 +1,404 @@
+
+### Run with Headed Browser
+```bash
+bun run test:e2e -- --headed
+```
+
+---
+
+## Test Fixtures
+
+Media fixtures are located at:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)